src/Access/CustomControllerAccessCallback.php

```diff
@@ -29,6 +29,11 @@ public function check(RouteMatchInterface $route_match): AccessResultInterface
                 $route_match->getRouteName()
             ));
         }
+        foreach ($access_arguments as &$argument) {
+            if (is_string($argument) && $placeholder = preg_filter('/(^{)(.*)(}$)/', '$2', $argument)) {
```

Comment:
Can you help explain this regex? I'm trying to parse it, but `_custom_access_arguments` is from `access arguments`. It should just be an `array_map` to get the values from the route match.

Edit: I read this before `src/Routing/HookMenuRoutes.php` changes.

Reply:
Right, I enclosed access arguments in curly braces if they need to be processed with a load function.